Assets/Shopify/UIToolkit/Editor/ShopCredentialsVerifier.cs

```diff
@@ -73,6 +73,9 @@ public class ShopCredentialsView : IShopCredentialsView {
     /// and drawing a UI to support the feature.
     /// </summary>
     public class ShopCredentialsVerifier {
+        public delegate void OnCredentialsStateChangedHandler();
+        public event OnCredentialsStateChangedHandler OnCredentialsStateChanged;
```

Comment:
This event is used to update the theme controllers client when the credentials change